docs/tutorials/connecting-to-the-internet-via-bluetooth.md

```diff
@@ -255,9 +255,54 @@ If you are using an older version, please upgrade.
 
         {% include /util/screenshot.html source="/images/brickman/networking-my-computer-0-disconnect-selected-online.png" %}
 
-*   {: tab="Windows"}__HELP WANTED!__ See [issue #287](https://github.com/ev3dev/ev3dev/issues/287).
+*   {: tab="Windows"}
+    These instructions were written using Windows 10 Enterprise.
+    It should work for other versions of Windows 10 as well.
+    {: .alert .alert-info}
+
+	1.	On the EV3, first verify that Bluetooth is powered on. In brickman,
+        open the *Wireless and Networks* menu and select *Bluetooth*. Make sure the
+        *Powered* checkbox is checked. The Bluetooth icon next to the battery in the
+        status bar also indicates that Bluetooth is powered on.
+		
+        {% include /util/screenshot.html source="/images/brickman/bluetooth-powered-visible-no-devices.png" %}
+	
+	2.	On your host computer, open the Action Center on the right end of the Taskbar and right-click on the Bluetooth Button.
+		Select:*Go to Settings*. Turn Bluethooth *On*
+		
+	3.	On the EV3, select *Start Scan* on the *Bluetooth* menu. It should find your
+        computer.
+
+        {% include /util/screenshot.html source="/images/brickman/bluetooth-powered-my-computer.png" %}
+		
+	4.  Select your computer from the menu and then select the *Pair* button. On the your host computer click on 
```

Comment:
It might be worth noting that the default name for Windows 10 PCs is a long string of random numbers and letters. You can actually look at the "Bluetooth" screen to see what name Windows is broadcasting under, so maybe you could just say "take note of the name..." above.